assets/css/campfire-games.css

```diff
@@ -0,0 +1,42 @@
+h2 {
+  margin-bottom: 20px;
+}
```

Comment:
```suggestion
```
If you want to keep it consistent to the main page, add the `.section-title` class to all your `<h2>`'s.